pkg/controller/podgroup.go

```diff
@@ -266,3 +276,17 @@ func (ctrl *PodGroupController) patchPodGroup(old, new *schedv1alpha1.PodGroup)
 	}
 	return nil
 }
+
+func fillOccupiedObj(pg *schedv1alpha1.PodGroup, pod *v1.Pod) {
+	refs := make([]string, 0)
+	for _, ownerRef := range pod.OwnerReferences {
+		refs = append(refs, fmt.Sprintf(pod.Namespace, ownerRef.Name))
```

Comment:
"fmt.Sprintf(pod.Namespace, ownerRef.Name)" looks weird, I guess you want to use a `/` separator?

Reply:
Yes, sorry for mistake